### PR TITLE
DEVDOCS-6141: BODL, update line_items fields

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-6141]

## What changed?

* Update BODL documentation's `line_items` field

## Release notes draft

* We're happy to announce the new `line_items.coupon_amount` field in Big Open Data Layer for hosted storefronts, which indicates the coupon amount for a line item in a cart.
* Fixed a bug in the Big Open Data Layer documentation regarding which BigCommerce API field to which the `line_items.discount` maps.

## Anything else?
@bigcommerce/team-data

[DEVDOCS-6141]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-6141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-rmalyavc — DISMISSED** [review]
Looks good 👍

File: docs/integrations/hosted-analytics.mdx [hunk 1]
```diff
@@ -208,8 +208,9 @@ The `line_item` object has many common fields for browser events, including the
...
 | `brand_name` | string | Brand name. | [Get a product](/docs/rest-catalog/products#get-a-product) <br /> `data.brand_name` |
 | `category_names[]` | array of strings | Category names. | [Get a product](/docs/rest-catalog/products#get-a-product) <br /> `data.categories[]` |
+| `coupon_amount` | number | The total value of all coupons applied to this item. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `cart.lineItems.physicalItems.couponAmount` or `cart.lineItems.digitalItems.couponAmount` |  
 | `currency` | string | [ISO-4217 currency code](https://www.iso.org/iso-4217-currency-codes.html) for the transaction. | [Get a checkout](/docs/rest-management/checkouts#get-a-checkout) <br />  `data.cart.currency.code`  |
-| `discount` | number | Discount applied to the purchase. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `data.cart.discountAmount` |
+| `discount` | number | Discount applied to the purchase. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `cart.lineItems.physicalItems.discountAmount` or `cart.lineItems.digitallItems.discountAmount` |
```
Comment (bc-traciporter) [inline docs/integrations/hosted-analytics.mdx:213]:
```suggestion
| `discount` | number | Discount applied to the purchase. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `cart.lineItems.physicalItems.discountAmount` or `cart.lineItems.digitalItems.discountAmount` |
```

**Review by bc-traciporter — COMMENTED** [review]
I found one typo

**Review by bc-traciporter — APPROVED** [review]
Approved

**Pull request merged**